packages/common/src/ui/components/inputs/DatePickers/DatePickerInput/DatePickerInput.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FC } from 'react';
+import { BaseDatePickerInput } from '../BaseDatePickerInput';
+
+interface DatePickerInputProps {
+  value: Date | null;
+  onChange: (value: Date | null) => void;
+}
+
+export const DatePickerInput: FC<DatePickerInputProps> = ({
+  value,
+  onChange,
+}) => {
+  return (
+    <BaseDatePickerInput
+      inputFormat="dd/MM/yyyy"
```

Comment:
I wonder is this affected if the browser settings are for a different locale 🤔 
Don't think you should either change or check it. I was just wondering if we'll have problems later on

Reply:
yes I think we will 😁